test/command/suite/select/column/stage/initial/reference_count.test

```diff
@@ -0,0 +1,30 @@
+#$GRN_ENABLE_REFERENCE_COUNT=yes
+table_create Items TABLE_HASH_KEY ShortText
+column_create Items price COLUMN_SCALAR UInt32
+
+table_create Prices TABLE_PAT_KEY UInt32
+column_create Prices item_index COLUMN_INDEX Items price
+
+load --table Items
+[
+{"_key": "Book",  "price": 1000},
+{"_key": "Note",  "price": 1000},
+{"_key": "Box",   "price":  520},
+{"_key": "Pen",   "price":  510},
+{"_key": "Food",  "price":  500},
+{"_key": "Drink", "price":  300}
+]
+
+#@add-important-log-levels dump
+#@add-ignore-log-pattern /\A\[io\]/
+log_level --level dump
+select Items \
+  --column[price_with_tax].stage initial \
```

Comment:
Could you use `columns` instead of `column` because `column` is deprecated?

Reply:
I have fixed it.